ztp/du-profile-kube-compare/metadata.yaml

```diff
@@ -0,0 +1,197 @@
+          - path: required/cluster-tuning/monitoring-configuration/ReduceMonitoringFootprint.yaml
+          - path: required/cluster-tuning/operator-hub/DefaultCatsrc.yaml
+          - path: required/cluster-tuning/operator-hub/DisconnectedICSP.yaml
+          - path: required/cluster-tuning/operator-hub/OperatorHub.yaml
+  - name: required-machine-config
+    Components:
+      - name: machine-config
+        type: Required
+        requiredTemplates:
+          - path: required/machine-config/disable-crio-wipe/99-crio-disable-wipe-master.yaml
+          - path: required/machine-config/disable-crio-wipe/99-crio-disable-wipe-worker.yaml
+          - path: required/machine-config/enable-cgroup-v1/enable-cgroup-v1.yaml
+          - path: required/machine-config/kubelet-configuration-and-container-mount-hiding/01-container-mount-ns-and-kubelet-conf-master.yaml
+          - path: required/machine-config/kubelet-configuration-and-container-mount-hiding/01-container-mount-ns-and-kubelet-conf-worker.yaml
+          - path: required/machine-config/one-shot-time-sync/99-sync-time-once-master.yaml
+          - path: required/machine-config/one-shot-time-sync/99-sync-time-once-worker.yaml
+          - path: required/machine-config/sctp/03-sctp-machine-config-master.yaml
+          - path: required/machine-config/sctp/03-sctp-machine-config-worker.yaml
+          - path: required/machine-config/set-rcu-normal/08-set-rcu-normal-master.yaml
+          - path: required/machine-config/set-rcu-normal/08-set-rcu-normal-worker.yaml
+          - path: required/machine-config/sriov-related-kernel-arguments/07-sriov-related-kernel-args-master.yaml
+          - path: required/machine-config/sriov-related-kernel-arguments/07-sriov-related-kernel-args-worker.yaml
+          - path: required/machine-config/crun/enable-crun-master.yaml
+          - path: required/machine-config/crun/enable-crun-worker.yaml
+  - name: required-node-tuning-operator
+    Components:
+      - name: node-tuning-operator
+        type: Required
+        requiredTemplates:
+          - path: required/node-tuning-operator/PerformanceProfile.yaml
+          - path: required/node-tuning-operator/TunedPerformancePatch.yaml
+  - name: required-ptp-operator
+    Components:
+      - name: ptp-operator
+        type: Required
+        requiredTemplates:
+          - path: required/ptp-operator/PtpConfigBoundary.yaml # TODO: the INI files embedded in PTPConfig are not handled. CNF-13528
+          - path: required/ptp-operator/PtpConfigGmWpc.yaml
+          - path: required/ptp-operator/PtpConfigSlave.yaml
+          - path: required/ptp-operator/PtpSubscription.yaml
+            config:
+              ignore-unspecified-fields: true
+          - path: required/ptp-operator/PtpSubscriptionNS.yaml
+          - path: required/ptp-operator/PtpSubscriptionOperGroup.yaml
+            config:
+              ignore-unspecified-fields: true
```

Comment:
This is the only *OperGroup with ignore-unspecified-fields. We should be consistent across all of them. For now none of the opergroup should ignore unspecified fields.

Reply:
removed ignore-unspecified-fields